extension/src/display.js

```diff
@@ -150,6 +150,10 @@ export class Binder {
                 link.title = 'Jump to the context';
                 // $FlowFixMe
                 link.href = loc.href;
+
+                // this seems to work only for the "editor://" protocol. avoids opening a new tab for the "editor://" link. http links then require a middle-click, which is undesirable..
```

Comment:
Oh, that's nice! I guess could check if the link is `editor://` and only set target to `_self` in that case?

Reply:
good idea, and seems to work too, at least in desktop firefox and chrome.